### PR TITLE
#90 set base url

### LINKED ISSUE #90: Change of host for url calls
Is there a way to change backend host for which every call is made? My situation is that I have different server for frontend and different server for backend. Backend server is on `3000` port and frontend app is on `9000` port.

### PR DESCRIPTION
**Pull request by TheOneTheOnlyDavidBrown**
closes #90

### CODE REVIEW

**Comment by azranel** [comment]
I think it would be better to config `AuthProvider` in `.config` than config it for `Auth`. This is setting of URL and you set login and other paths also for `AuthProvider`.

**Comment by TheOneTheOnlyDavidBrown** [comment]
@azranel I agree. I made a faulty assumption when making the change that the $get methods could be accessed in the AuthProvider. Changes pushed and tested in a local instance

File: src/auth.js [hunk 1]
```diff
@@ -50,7 +51,7 @@ devise.provider('Auth', function AuthProvider() {
     function httpConfig(action, data, additionalConfig) {
         var config = {
             method: methods[action].toLowerCase(),
-            url: paths[action]
+            url: _baseUrl+''+paths[action]
```
Comment (azranel) [inline src/auth.js]: Why you add this empty string in the middle? :smile:

**Comment by azranel** [comment]
You can also check #92 PR. Tried to make it as simple as possible.

**Comment by TheOneTheOnlyDavidBrown** [comment]
closing this because #92 has tests and is a little cleaner

**Pull request closed without merging**